crates/muvm/src/guest/bin/muvm-guest.rs

```diff
@@ -57,7 +57,7 @@ fn main() -> Result<()> {
 
     configure_network()?;
 
-    if let Some(hidpipe_client_path) = find_in_path("hidpipe-client")? {
+    if let Some(hidpipe_client_path) = find_in_path("muvm-hidpipe")? {
```

Comment:
Now thta we're bundling, we should drop this check.

Reply:
:-/ Seems like a packaging issue if we can't find it though. would be inclined to make this a hard req instead of getting weird "gamepads don't work on Ubuntu Asahi help!!" bugs when it could've been caught earlier on

Reply:
The flip side is that not everyone uses gamepads / controllers, and we have already seen in some issues here that there are people who want to run muvm headless. So I think it actually makes sense to have this feature gated.

Reply:
Is it really necessary to feature gate it at the source level though? Distros probably aren't going to split-package this, it's  kind of pointless... for people who don't use gamepads/controllers, it should just do nothing at runtime.

Reply:
I agree with @asahilina here. Gating stuff has a maintenance cost, so there must be a benefit that justifies it, and I don't think enabling distros to disable hidpipe does it. In fact, I think x11bridge should be ungated as soon as it's made the default (which, based on my experience testing it, should be done fairly soon).

I'm going to change the `find_in_path` with a direct `Command::new()`, as we're doing with `muvm-server` and @alyssarosenzweig suggested.